.dockerignore

```
@@ -0,0 +1 @@
+node_modules
```

👍

Should be able to expand on it too:

```suggestion
/.docker/
/.github/
/.scripts/
/build/
/Makefile
/node_modules/
```